style/gallery.less

```diff
@@ -60,27 +58,24 @@
 
 	&-main {
 		display: flex;
-		display: -ms-flex;
 		max-width: 100%;
 		height: 100%;
 	}
 
 	&-slider {
-		display: inline-flex;
-		display: -ms-inline-flex;
+		display: flex;
```

Comment:
Hopefully these won't conflict too much with #85 or #86. Working well in Firefox and Chrome for me right now!

Reply:
good news 🎉 I have tested merging the others (only 1 conflict which easy to fix), works fine 💯